plugin.php

```diff
@@ -8,6 +8,14 @@
  * License: GPL2+
  * License URI: https://www.gnu.org/licenses/gpl-2.0.txt
  */
+function a8c_meeting_calendar_callback( $attributes, $content ) {
```

Comment:
I think we should switch to using `wporg_` prefixes, since this is primarily for Meta.

Reply:
Okay, i'll make the update. In the near future. I'm closing this PR in favor of:

https://github.com/Automattic/meeting-calendar/pull/6